hazelcast/src/main/java/com/hazelcast/internal/dynamicconfig/DynamicConfigurationAwareConfig.java

```diff
@@ -243,7 +243,7 @@ public Config setMapConfigs(Map<String, MapConfig> mapConfigs) {
     }
 
     @Override
-    public CacheSimpleConfig findCacheConfig(String name) {
+    public CacheSimpleConfig findCacheConfigOrNull(String name) {
```

Comment:
Doesn't this change the behaviour of the `Config.findCacheConfig(name)` for a configuration obtained from a running hazelcast instance? @vbekiaris is this acceptable?